actions/hls_memcopy_1024/README.md

```diff
@@ -7,5 +7,5 @@
   * Block RAM memory (memory inside the FPGA itself)
 * The example code details the copy mechanism.
 
-:star: Please check the [actions/hls_memcopy/doc](./doc/) directory for detailed information
+:star: Please check the [actions/hls_memcopy/_1024doc](./doc/) directory for detailed information
 
```

Comment:
This documentation has been moved to https://opencapi.github.io/oc-accel-doc/actions-doc/hls_memcopy_1024/

Reply:
corrected for memcopy 1024 and 512...
Will need to create 512 doc and correct 1024 doc with right numbers
Thx